index.html

```diff
@@ -0,0 +1,276 @@
+              <div class="people-info">
+                <h3 class="speaker">Guillermo Velazquez</h3>
+                <h4 class="red-subtitle italic">Mayor</h4>
+                <p class="info">
+                  The Mayor of this town is the one who proposed that the event
+                  be held in this municipality. All this magic is thanks to the
+                  fact that he decided to lend this municipality.
+                </p>
+              </div>
+            </div>
+
+            <div class="cards">
+              <img src="Images/sandra.jpg" alt="speaker3" />
+              <div class="people-info">
+                <h3 class="speaker">Sandra Lopez</h3>
+                <h4 class="red-subtitle italic">Event Organizer</h4>
+                <p class="info">
+                  Sandra is the only person in the world that can have a lot of
+                  people under her arm. She has an incredible talent to organice
+                  every single area to perfection. Without her at the helm, this
+                  event would be known for its chaos.
+                </p>
+              </div>
+            </div>
+          </section>
+          <section class="column">
+            <div class="cards">
+              <img src="Images/vaquero.jpg" alt="speaker4" />
+              <div class="people-info">
+                <h3 class="speaker">Arturo Ramirez</h3>
+                <h4 class="red-subtitle italic">Cowboy</h4>
+                <p class="info">
+                  Miguel is the most coveted and qualified cowboy of the state.
+                  He is the only one capable of making a group of horses dance
+                  in tune.
+                </p>
+              </div>
+            </div>
+
+            <div class="cards">
+              <img src="Images/poncho.jpg" alt="speaker5" />
+              <div class="people-info">
+                <h3 class="speaker">Alonso Mora</h3>
+                <h4 class="red-subtitle italic">Show Organizer</h4>
+                <p class="info">
+                  Alonso is the best show organizer of the country· Big
+                  companies have request his organizer services because they
+                  know that his shows that he prepares are incredible. You will
+                  never get bored in one of his shows.
+                </p>
+              </div>
+            </div>
+
+            <div class="cards">
+              <img src="Images/amram2.jpg" alt="speaker6" />
+              <div class="people-info">
+                <h3 class="speaker">Amram Martinez</h3>
+                <h4 class="red-subtitle italic">Security</h4>
+                <p class="info">
+                  Amram is the responsable of all the security system. Having a
+                  military training is his powerful weapon. His team is prepared
+                  for all kinds of mishaps.
+                </p>
+              </div>
+            </div>
```

Comment:
The **height** of the text content, **alignment** and **typography**  of your _hero_ / _introduction_ section do not match the provided [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). We actually want our design to as much as possible closely mimic the design of the provided template. Kindly ensure to improve upon these:
- [ ] The typography (**font family**, **font sizes**, **font-weight**, **font color**) of the text content should match the provided design template.
- [ ] The height of the text content should match the height of the image of the height, as can be seen in the provided template design.
- [ ] Contents of this section should be **evenly spaced** (paddings/ margins), as can be seen in the provided template design.
- [ ] The images should have the **same dimensions**, and the dimensions should match the provided template design.
- [ ] Add **background decorations** to the images with missing background decorations

**Your hero design:**
![image](https://user-images.githubusercontent.com/12745474/133910220-a310e555-67cb-4869-beb4-b895007495e3.png)


**The original hero design:**
![image](https://user-images.githubusercontent.com/12745474/132607606-41b0af31-fa67-4194-8d5d-5874348ab4d8.png)

Reply:
> The **height** of the text content, **alignment** and **typography** of your _hero_ / _introduction_ section do not match the provided [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). We actually want our design to as much as possible closely mimic the design of the provided template. Kindly ensure to improve upon these:
> 
> * [x]  The typography (**font family**, **font sizes**, **font-weight**, **font color**) of the text content should match the provided design template.
> * [x]  The height of the text content should match the height of the image of the height, as can be seen in the provided template design.
> * [x]  Contents of this section should be **evenly spaced** (paddings/ margins), as can be seen in the provided template design.
> * [x]  The images should have the **same dimensions**, and the dimensions should match the provided template design.
> * [x]  Add **background decorations** to the images with missing background decorations
> 
> **Your hero design:**
> ![image](https://user-images.githubusercontent.com/12745474/133910220-a310e555-67cb-4869-beb4-b895007495e3.png)
> 
> **The original hero design:**
> ![image](https://user-images.githubusercontent.com/12745474/132607606-41b0af31-fa67-4194-8d5d-5874348ab4d8.png)

- A wise man once said if people pay attention to what you do and provide constructive suggestions on how to improve and make it better, you must never let them out of site 😉 . Do well to handle this amazing suggestion it will help significantly improve our work 👍